app/src/main/java/com/fstyle/structure_android/data/source/remote/UserRemoteDataSource.java

```diff
@@ -0,0 +1,55 @@
+package com.fstyle.structure_android.data.source.remote;
+
+import com.fstyle.structure_android.data.model.User;
+import com.fstyle.structure_android.data.source.UserDataSource;
+import com.fstyle.structure_android.data.source.remote.config.response.SearchUserResponse;
+import com.fstyle.structure_android.data.source.remote.config.service.NameApi;
+import com.fstyle.structure_android.data.source.remote.config.service.NameServiceClient;
+import io.reactivex.Maybe;
+import io.reactivex.Single;
+import io.reactivex.SingleSource;
+import io.reactivex.functions.Function;
+import java.util.List;
+
+/**
+ * Created by daolq on 1/10/18.
+ */
+
+public class UserRemoteDataSource implements UserDataSource.RemoteDataSource {
+
+    private static UserRemoteDataSource INSTANCE;
```

Comment:
WARNING: [SuspiciousConstantFieldName] The field name indicates a constant but its modifiers do not
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/naming.html#SuspiciousConstantFieldName